ddp-workspace/projects/ddp-singular/src/app/app.component.scss

```diff
@@ -1,7 +1,26 @@
+@import '../styles/variables';
+@import '../styles/breakpoints';
+
 :host {
   display: flex;
   flex-flow: column nowrap;
-  height: 100%;
+  min-height: 100vh;
```

Comment:
The bug root cause is here. 
Scrolling to the first invalid/empty required field was not working properly because 
**window.scrollY** property returned 0 
(`ddp-sdk\src\lib\components\activityForm\activity-blocks\activityQuestion.component.ts` , 
 `setupScrollToErrorAction` method)
 
![n1](https://user-images.githubusercontent.com/7396837/182885873-d2a32d9a-ede3-4a2a-8792-23774b2b6f60.png)

Similar issue on [stackoverflow](https://stackoverflow.com/questions/33447504/window-scrolly-not-working#:~:text=you%20need%20to,should%20work%20then.).